backends/tfhe-cuda-backend/cuda/src/pbs/bootstrap_low_latency.cu

```diff
@@ -530,6 +379,41 @@ void cuda_bootstrap_low_latency_lwe_ciphertext_vector_32(
   }
 }
 
+/* Perform bootstrapping on a batch of input u32 LWE ciphertexts.
+ * This function performs best for small numbers of inputs. Beyond a certain
```

Comment:
This comment is outdated, maybe we can update such comments in the renaming PR.

Reply:
Indeed. I will write a note to do that in the next PR.